go/worker/compute/executor/committee/orderedmap/ordered_map.go

```diff
@@ -0,0 +1,194 @@
+// Package orderedmap implements a queue backed by an ordered map.
```

Comment:
Should this instead go somewhere under `go/runtime/scheduling`?

Reply:
Yeah good idea, i'll move it there.